libs/sdk-ui-dashboard/api/sdk-ui-dashboard.api.md

```diff
@@ -3784,6 +3814,9 @@ export const selectStash: OutputSelector<DashboardState, Record<string, Extended
 // @alpha
 export const selectUser: OutputSelector<DashboardState, IUser, (res: UserState) => IUser>;
 
+// @alpha
+export const selectUserRef: OutputSelector<DashboardState, ObjRef, (res: IUser) => ObjRef>;
+
```

Comment:
what user is this? creator? owner? current user?

Reply:
Fixed renamed to selectCurrentUserRef